.github/workflows/PR.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         with:
           api-level: 29
           arch: x86_64
-          script: ./gradlew connectedCheck :sqldelight-gradle-plugin:build --stacktrace -PInstrumentation
+          script: ./gradlew connectedCheck :sqldelight-gradle-plugin:instrumentationTest --stacktrace
```

Comment:
isn't this doing the same thing? the Instrumentation annotation was meant to allow partitioning of tests across two different shards

Reply:
Yeah, but instead it runs all tests including `Instrumentation` instead running only `Instrumentation`.

Reply:
I think thats only true for `grammarKitTest` and `dockerTest` because they dont specify [category rules](https://github.com/cashapp/sqldelight/blob/master/sqldelight-gradle-plugin/build.gradle#L134)

Reply:
I'm not strongly pushing back, its probably fine, but the gradle plugin gradle file is getting gnarly and I'm trying to see if there are ways to make it more sane. Each test task explicitly adding the local publishing stuff i think is the worst bit

Reply:
Yeah, I looked at the executed test results at this moment... Now: https://scans.gradle.com/s/dzucdtotn7ivw/tests/overview vs https://scans.gradle.com/s/o3ncaukzzcmtu/tests/overview
Why are the integration tests so slow?

Reply:
> i think is the worst bit

Regarding developer ux? Yeah, but doing integration testing with Gradle and project modules is pain. In another project, I added the parent project as included build to the test projects, but I don't think, this is really better. 

But I am still shocked seeing the times required for 1 test, it's almost 2 minutes just to execute the app on the virtual device. I didn't expect the test to do only 7 tests within the same time as all normal Gradle tests.